apps/linows/src/css/theme.css

```
@@ -119,6 +140,49 @@
+    --ext-script: rgb(122, 96, 12);
+    --ext-lang: rgb(150, 58, 42);
+    --ext-style: rgb(112, 51, 108);
+    --ext-data: rgb(30, 88, 104);
+
+    --syntax-keyword: rgb(112, 51, 108);
+    --syntax-string: rgb(133, 89, 8);
+    --syntax-comment: rgb(122, 116, 105);
+    --syntax-number: rgb(30, 88, 104);
+
+    --font-family:
+        "Charter", "Bitstream Charter", "Charis SIL", "Literata", Georgia, "Noto Serif",
```

Stylelint (17.14.1)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)